source/material_model/diffusion_dislocation.cc

```diff
@@ -336,43 +409,55 @@ namespace aspect
                              "viscosity at that point.  Select a weighted harmonic, arithmetic, "
                              "geometric, or maximum composition.");
           // Diffusion creep parameters
-          prm.declare_entry ("Prefactors for diffusion creep", "1.92e-11",
+          prm.declare_entry ("Prefactors for diffusion creep", "1.5e-15",
```

Comment:
The old model before the patch gave the incorrect answer (for the same reasons as the Morency Doin model), so I took the opportunity to change the parameters to something more relevant.